### PR TITLE
Add VO and topology info for hosting IceCube StashCache data at CHTC (FD #10141)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
@zvada does this look good to you too?

File: virtual-organizations/IceCube.yaml [hunk 1]
```diff
@@ -49,3 +49,14 @@ PrimaryURL: https://grid-voms.desy.de:8443/voms/icecube/
...
 ReportingGroups:
 - IceCube
+
+DataFederations:
+  StashCache:
+    Namespaces:
+      /icecube/PUBLIC:
+        - ANY
+    AllowedCaches:
+      - ANY
```
Comment (brianhlin) [inline virtual-organizations/IceCube.yaml]: Did you run this by Benedikt?

Comment (matyasselmeci) [reply virtual-organizations/IceCube.yaml]: I guess I should ask if he wants to make the PUBLIC/PROTECTED split or not...

Comment (matyasselmeci) [reply virtual-organizations/IceCube.yaml]: Benedikt agrees with the split.

**Comment by matyasselmeci** [comment]
Merging this for now. Let me know if there's something that needs fixing.

**Pull request merged**